assets/prototype/application/utilities/money/parsedAmount.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * returns amount parsed as a float (if not already a number)
+ *
+ * @param {number|string} amount
+ * @return {number}
+ */
+const parsedAmount = (amount: number | string): number => {
```

Comment:
When can an amount be a `string`?

Reply:
when it comes from react final form

Reply:
I like the `strict typing` idea, I think it will ease the understanding and removes away the mental overload related to coercion.